delete-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusDeleteHandlerTest.java

```diff
@@ -91,17 +90,11 @@ void shouldEmitEventsWithScopusIdentifiersWhenInputFileContainsMultipleScopusIde
         var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
         var s3Event = createS3Event(uri);
         scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
-        var actualScopusDeleteEventBodies = getScopusDeleteEventBodies();
-        assertThat(actualScopusDeleteEventBodies.size(),
-                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.size())));
-        assertThat(actualScopusDeleteEventBodies.get(0).getScopusIdentifier(),
-                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(0))));
-        assertThat(actualScopusDeleteEventBodies.get(1).getScopusIdentifier(),
-                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(1))));
-        assertThat(actualScopusDeleteEventBodies.get(2).getScopusIdentifier(),
-                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(2))));
-        assertThat(actualScopusDeleteEventBodies.get(3).getScopusIdentifier(),
-                is(equalTo(HARDCODED_SCOPUS_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.get(3))));
+        var actualIdentifiersFromEventBodies = getIdentifiersFromEventBodies();
+        assertThat(actualIdentifiersFromEventBodies.size(),
```

Comment:
Here is where I will likely differ from others' advice — if we compare two Java Sets directly using JUnit's assertEquals, we cover both of these tests with a single test.

Reply:
Changed to assertEquals and removed unnecessary size check